drivers/snmp-ups.c

```diff
@@ -392,6 +392,19 @@ void upsdrv_initups(void)
 		dstate_addcmd("shutdown.return");
 		dstate_addcmd("shutdown.stayoff");
 	}
+
+	/* Publish sysContact and sysLocation for all subdrivers */
+	/* sysContact.0 */
+	if (nut_snmp_get_str(".1.3.6.1.2.1.1.4.0", model, sizeof(model), NULL) == TRUE)
```

Comment:
not sure for macros, since I've tried to get rid of this in implementation files (*-mib.c).
When doing the present PR, I was also hesitating between the central point (as done), since it's worth for all subdrivers, Vs adding to each subdriver and get that extracted for DMFs.
Each has pros and cons.
That said, it would be good for DMF to be able to have some base.dmf that could then be included in others, but that implies other things (not all .dmf to be parsed, ...)
As for extended attributes, these specific OIDs have no particular default values and are only useful for customers / users to push their own info into it.